### ISSUE TITLE
Splash texts on subgame

### ISSUE THREAD

**Issue by WSDguy2014** [Wuzzy2/MineClone2-Bugs#59]
Do you played game Magichet (a sandbox game based on Minetest engine) before? It's also has some Minecraft features that made by 4aiman. You can try out first by googling it and comment.

**Comment by Wuzzy2** [comment]
I'm not really sure if I want this.

**Comment by tobyplowy** [comment]
@Wuzzy2 i wuld also like this but i dount know if its possible (is it posible to edit the menu?)

**Comment by WSDguy2014** [comment]
@tobyplowy, just ask 4aiman.

**Comment by Wuzzy2** [comment]
Yes, it is. You can define alternative images and Minetest picks one randomly each start. I just have to do this with the text logo.
I just don't like this approach because I would need a new image file for each new silly caption. A clean solution would be if it would be possible to add real texts (not images) on the title screen.
